airbyte-integrations/singer/exchangeratesapi/source/source_implementation.py

```diff
@@ -0,0 +1,27 @@
+from source import Source
+from integration import AirbyteSpec
+from integration import AirbyteCheckResponse
+from integration import AirbyteSchema
+from integration import AirbyteMessage
+import urllib.request
+from typing import Generator
+from singer_helpers import SingerHelper
+
+
+class SourceImplementation(Source):
```

Comment:
i wonder if we can set default so that if your singer integration does all of the "normal" stuff that you don't need to write any code here except the name of the singer tap executable itself. don't think we should act on this now, but maybe worth keeping in mind.

we might do this by having default implementations for these methods or a `DefaultSourceImplementation` class that just takes in a name.

Reply:
Sounds good, I'll add separately.